app/models/trainee.rb

```diff
@@ -169,6 +174,16 @@ class Trainee < ApplicationRecord
   scope :with_manual_application, -> { where(apply_application: nil) }
   scope :with_apply_application, -> { where.not(apply_application: nil) }
 
+  scope :on_early_years_routes, -> { where(training_route: EARLY_YEARS_TRAINING_ROUTES.keys) }
+
+  scope :with_education_phase, lambda { |*levels|
+    education_phases = levels.reject { |level| level == "early_years" }
```

Comment:
this `early_years` string feels like it should be defined somewhere. We've used it a few times to compare against